sigopt/run_params.py

```diff
@@ -0,0 +1,130 @@
+from collections.abc import MutableMapping
+
+from .lib import is_string
+
+
+_get = object.__getattribute__
+_set = object.__setattr__
+
+class RunParameters(MutableMapping):
+  def __init__(self, run_context, fixed_items):
+    _set(self, "__items", dict(fixed_items))
+    _set(self, "__run_context", run_context)
+    _set(self, "__fixed_keys", set(fixed_items.keys()))
+
+  def update(self, *args, **kwds):  # pylint: disable=arguments-differ
```

Comment:
`kwargs` instead of `kwds`? I've never seen `kwds` before

Reply:
https://github.com/python/cpython/blob/main/Lib/_collections_abc.py#L983

Reply:
the signature in this file is not compatible with older versions of python (`update(self, other=(), /, **kwds)`)

Reply:
ah, ok